cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/ItalianElectronicInvoicingDTO.java

```diff
@@ -0,0 +1,59 @@
+package com.cloudesire.platform.apiclient.dto.model.dto;
+
+import com.liberologico.cloudesire.common.Regexp;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.Size;
+import java.util.Objects;
+
+@ApiModel( description = "Electronic invoicing data for Italian business companies" )
+public class ItalianElectronicInvoicingDTO
+{
+    @ApiModelProperty( "Codice univoco ufficio" )
+    @Size( min = 6, max = 7 )
+    private String cuu;
```

Comment:
use a more generic name since cuu is the acronym of one of the two possibile things to put here